ports/stm32/main.c

```diff
@@ -88,6 +87,15 @@
 #include "pyb_can.h"
 #include "subghz.h"
 
+#if MICROPY_HW_TINYUSB_STACK
+#include "usbd_conf.h"
+#include "shared/tinyusb/mp_usbd.h"
+#endif
+
+#if MICROPY_HW_ENABLE_USB
+#include "usb.h"
```

Comment:
Couldn't we just unconditionally include this particular header, like it was before this PR?  Keeps it simple.